test/testcafe/spec/TerminalView_spec.js

```diff
@@ -302,3 +322,62 @@ test.requestHooks(endUserRemediationNoOptionsMock)('should render end user remed
 });
+
+test.requestHooks(endUserRemediationMultipleOptionsWithCustomRemediationMock)('should render end user remediation when there are multiple options with custom messages and URLs', async t => {
+  const terminalViewPage = await setup(t);
+  await checkA11y(t);
+
+  await t.expect(terminalViewPage.form.getErrorBoxCallout().withText('Your device doesn\'t meet the security requirements').exists).eql(true);
+  await t.expect(terminalViewPage.form.getErrorBoxCallout().withText('To sign in, pick an option and make the updates. Then, access the app again.').exists).eql(true);
+
+  await t.expect(terminalViewPage.form.getErrorBoxAnchor('https://myremediationurl.com/docs').withExactText('It\'s our company policy that you upgrade your Android OS').exists).eql(true);
+  await t.expect(terminalViewPage.form.getErrorBoxAnchor('https://okta.com/android-biometric-lock').withExactText('Enable lock screen and biometrics').exists).eql(true);
+
+  await t.expect(terminalViewPage.form.getErrorBoxAnchor('https://okta.com/android-lock-screen').withExactText('Enable lock screen').exists).eql(true);
+  await t.expect(terminalViewPage.form.getErrorBoxCallout().withText('It\'s our company policy that your Android device cannot be rooted').exists).eql(true);
+
+  await t.expect(terminalViewPage.form.getErrorBoxCallout().withText('follow the instructions on the help page').find('a[href="https://okta.com/help"]').exists).eql(true);
+  await t.expect(terminalViewPage.form.getAnchorsWithBlankTargetsWithoutRelevantAttributes().exists).eql(false);
+});
+
+test.requestHooks(endUserRemediationCustomMessageCustomUrlMock)('should render end user remediation when there is an option with a custom message and custom URL', async t => {
+  const terminalViewPage = await setup(t);
+  await checkA11y(t);
+
+  await t.expect(terminalViewPage.form.getErrorBoxCallout().withText('Your device doesn\'t meet the security requirements').exists).eql(true);
+  await t.expect(terminalViewPage.form.getErrorBoxCallout().withText('To sign in, make the following updates. Then, access the app again.').exists).eql(true);
+
+  await t.expect(terminalViewPage.form.getErrorBoxAnchor('https://myremediationurl.com/docs').withExactText('It\'s our company policy that you upgrade your Android OS').exists).eql(true);
+  await t.expect(terminalViewPage.form.getErrorBoxAnchor('https://okta.com/android-biometric-lock').withExactText('Enable lock screen and biometrics').exists).eql(true);
+
+  await t.expect(terminalViewPage.form.getErrorBoxCallout().withText('follow the instructions on the help page').find('a[href="https://okta.com/help"]').exists).eql(true);
+  await t.expect(terminalViewPage.form.getAnchorsWithBlankTargetsWithoutRelevantAttributes().exists).eql(false);
+});
+
+test.requestHooks(endUserRemediationCustomMessageNoUrlMock)('should render end user remediation when there is an option with a custom message and no URL', async t => {
```

Comment:
Consider adding an assertion that there is no link. I realize a negative assertion is tricky, so I am looking for an assertion not on a specific link href missing, but that there is no anchor element or similar.